onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -7736,17 +7736,29 @@ def aten_scalar_tensor_sym_number(
     return common_ops.cast_to(s, dtype=dtype)
 
 
-@torch_op(("aten::scatter.value", "aten::scatter.src"), trace_only=True)
-def aten_scatter(
+@torch_op("aten::scatter.src", trace_only=True)
+def aten_scatter_src(
     self: TReal,
```

Comment:
One thing I just noticed:
 
```suggestion
    self: TensorType,
```

We should update the accepted type to make it more permissive. Could you create a follow PR?